build.gradle

```diff
@@ -13,6 +13,8 @@ buildscript {
         springBootVersion = "2.2.1.RELEASE"
         springfoxSwaggerVersion = "2.9.2"
         storeApiVersion = "0.3.0"
+        //TODO This is a placeholder. This version does not define a status endpoint.
+        transformApiVersion = "0.2.0"
```

Comment:
✏️ Add a dependency for this (compileOnly) so that dependabot will keep this version up to date

Reply:
`compileOnly "com.connexta.transformation:transformation-api-rest-spring-stubs:${transformApiVersion}"`